native/src/main/java/io/ballerina/stdlib/grpc/protobuf/cmd/GrpcCmd.java

```diff
@@ -293,12 +302,13 @@ private void generateBalFile(String protoPath) {
         BallerinaFileBuilder ballerinaFileBuilder;
         // If user provides output directory, generate service stub inside output directory.
         if (balOutPath == null) {
-            ballerinaFileBuilder = new BallerinaFileBuilder(root, dependant);
+            ballerinaFileBuilder = new BallerinaFileBuilder(root.getDescriptor(), getDependentDescriptorSet(dependant));
```

Comment:
Shall we pass DescriptorMeta objects directly into the `BallerinaFileBuilder` and compute `unusedImports` map inside build method

Reply:
Updated in [673fbdf](https://github.com/ballerina-platform/module-ballerina-grpc/pull/739/commits/673fbdf3937245c3406b7258b82810dfce3b25f6)